tests/NLog.UnitTests/LayoutRenderers/Wrappers/CachedTests.cs

```diff
@@ -119,11 +119,11 @@ public void CacheKeyTest()
         [Fact]
         public void CachedSecondsTimeoutTest()
         {
-            SimpleLayout l = "${guid:cachedSeconds=10}";
+            SimpleLayout l = "${guid:cachedSeconds=60}";
```

Comment:
why does the test need a change? 

Not that I'm against it, but don't see the reason for it.

Reply:
It was easier to debug, when having 60 secs instead of 10 secs.